src/main/java/io/confluent/connect/hdfs/parquet/ParquetRecordWriterProvider.java

```diff
@@ -49,7 +49,7 @@ public String getExtension() {
 
     Path path = new Path(fileName);
     final ParquetWriter<GenericRecord> writer =
-        new AvroParquetWriter<>(path, avroSchema, compressionCodecName, blockSize, pageSize);
+        new AvroParquetWriter<>(path, avroSchema, compressionCodecName, blockSize, pageSize, true, conf);
```

Comment:
I am also not sure if `true` here must be configurable. This is documentation for the parameter: `* @param enableDictionary Whether to use a dictionary to compress columns.`.
For our use case we use dictionary so code above fits our purpose nicely.